src/App.tsx

```diff
@@ -1,24 +1,106 @@
-import React, { FC } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { FC, useState } from 'react';
+import {
+  Box,
+  Button,
+  Container,
+  CssBaseline,
+  TextField,
+} from '@material-ui/core';
+import Todo from './components/Todo/Todo';
+import { Todo as ITodo } from './components/Todo/typings';
```

Comment:
I suggest creating a single import inside `Todo` folder creating a `index.ts`, ex.

```index.ts
export { default } from './Todo';
export { Todo } from './typings';
```

```suggestion
import Todo, { Todo as ITodo }  from './components/Todo';
```

Reply:
I took the suggestion but I faced naming colliding issues between the component and the interface homonymous. I could add a rule on TS config but I prefer to export the interface as `ITodo` from `components/Todo/index.ts`.
Resolved on ac40b7bd5b1418375c16876812c614ac7cb7f273